runelite-client/src/main/java/net/runelite/client/plugins/textrecolor/TextRecolorConfig.java

```diff
@@ -0,0 +1,589 @@
+package net.runelite.client.plugins.textrecolor;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+import java.awt.*;
+
+@ConfigGroup(
+	keyName = "textrecolor",
+	name = "Chat Text Recolor",
+	description = "Configuration for chat text recoloring"
+)
+public interface TextRecolorConfig extends Config
+{
+	@ConfigItem(
+		position = 30,
+		keyName = "__dummyTitleOpaque",
+		name = "Opaque Chat Colors",
+		description = ""
+	)
+	default long __dummyTitleOpaque()

```

Comment:
What is this?

Reply:
I'm using config options with invalid types to make an option without a value to use as a header.
For Example: the top value.
![java_2018-05-02_08-50-54](https://user-images.githubusercontent.com/2979691/39762261-11b23ec6-52d2-11e8-9d7b-2336f6c1bcde.png)

Reply:
I would prefer if we didn't do this